src/errors.rs

```diff
@@ -157,24 +154,14 @@ where
     /// kernel.
     #[error("empty access-right")]
     Empty,
-    /// The access-rights set was forged with the unsafe `BitFlags::from_bits_unchecked()` and it
-    /// contains unknown bits.
-    #[error("unknown access-rights (at build time): {unknown:?}")]
-    Unknown {
-        access: BitFlags<T>,
-        unknown: BitFlags<T>,
-    },
```

Comment:
This error is no longer possible as the public api of `Access*` ensures no unknown bits are set.